string object_name
geometry_msgs/PointStamped stamped_point
std_msgs/Float32 theta
int32[] pos_x
int32[] pos_y

